process.tex

```diff
@@ -1,42 +1,55 @@
 \section{Process} \label{sec:process}
 
+Throughout the Rubin pre-operations and survey operations phases, annually in May each team will look back at what was planned, what was achieved, do a full review of its activities, and propose a high level plan for the following (US fiscal) year.
 
-Throughout the Rubin pre-operations and operations phases, annually in May each team will look back at what was planned, what was achieved, do a full review of its activities, and propose a high level plan for the following (US fiscal) year.
+Such an ``annual scrubbing'' is standard practice in other high energy physics experiments. 
+The scrub allows the facility to continuously evolve its operating plan, taking critical input from the people that understand best what is really needed. 
+In Rubin’s case that is the Team Leaders, who are responsible for delivery at Level 3 of the observatory's work breakdown structure.
```

Comment:
we haven't really introduced WBS or level 3 as a term anywhere. I'll update this to responsible for delivering products.